xs/src/libslic3r/Fill/Fill.cpp

```diff
@@ -80,26 +76,27 @@ Fill::adjust_solid_spacing(const coord_t width, const coord_t distance)
 {
     assert(width >= 0);
     assert(distance > 0);
-    // floor(width / distance)
-    coord_t number_of_intervals = floor(width / distance);
-    coord_t distance_new = (number_of_intervals == 0)
-        ? distance
-        : (width / number_of_intervals);
+    const int number_of_intervals = floor(width / distance);
+    if (number_of_intervals == 0) return distance;
+    
+    coord_t distance_new = (width / number_of_intervals);
     
     const coordf_t factor = coordf_t(distance_new) / coordf_t(distance);
     assert(factor > 1. - 1e-5);
     
     // How much could the extrusion width be increased? By 20%.
     const coordf_t factor_max = 1.2;
-    if (factor > factor_max)
+    if (false && factor > factor_max)
```

Comment:
Ah right, @bubnikv added that and I disabled it temporarily while debugging. Will remove that `false`.